src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_constructs/adf_codepipeline.py

```diff
@@ -91,6 +92,22 @@ def _generate_configuration(self): #pylint: disable=R0912, R0911, R0915
                                     'properties', {}).get(
                                         'object_key')
             }
+        if self.provider == "CodeStarSourceConnection":
+            owner = self.map_params.get('default_providers', {}).get('source').get('properties', {}).get('owner', {})
+            repo = self.map_params.get('default_providers', {}).get('source', {}).get('properties', {}).get('repository', {}) or self.map_params['name']
+            codestar_connection_path = self.map_params.get('default_providers', {}).get('source').get('properties', {}).get('codestar_connection_path', {})
+            ssm_client = boto3.client('ssm')
+            try:
+                response = ssm_client.get_parameter(Name=codestar_connection_path)
+            except Exception as e:
+                print(f"No parameter found at {codestar_connection_path}. Check the path/value.")
```

Comment:
Could you change this print to use the logger instead. Like we do [here](https://github.com/awslabs/aws-deployment-framework/blob/31f4719f83e80bd165b1567fed27de8eff090460/src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_constructs/adf_notifications.py#L20-L25)? But using error instead.